inst/rmarkdown/templates/nutrition/skeleton/skeleton.Rmd

```diff
@@ -129,9 +149,18 @@ Prevalence of acute malnutrition based on weight-for-height z-scores (and/or oed
 ```{r}
 # we can either estimate the prevalence using simple couting of the z-scores
 # or use anthro_prevalence to use the survey package to obtain better estimates
+
+# Calculate denominators for each group
+totals <- c(length(survey_data$sex), sum(survey_data$sex==1), sum(survey_data$sex==2))
+
 dplyr::bind_cols(survey_data, weight_for_height) %>%
```

Comment:
not sure if this type of stuff (and rest of it below) - needs to go in to functions? Feel like people going to be put off by how much needs to go down before a table comes out. 
Building on dirks survey functions should simplify a lot of this. 
thoughts @zkamvar ?